nautobot/core/apps/__init__.py

```diff
@@ -182,6 +182,17 @@ def register_menu_items(tab_list):
 
                     registry_buttons = registry_groups[group.name]["items"][item.link]["buttons"]
                     for button in item.buttons:
+                        try:
+                            button.link = reverse(button.link, args=button.args, kwargs=button.kwargs)
```

Comment:
I'm not a big fan of modifying the registry item in place here (notably it causes exceptions if you try and re-register the same `menu_items` object a second time, e.g. in manual testing) but I know this is just replicating the pattern we already have for menu items above, so not blocking at this time.